public/stylesheet.css

```
@@ -536,7 +538,7 @@ input[type='search']::-ms-reveal {
 }
 
 /* Control - order */
-body.order-alphabetically #order-alpha,
+body.order-alpha #order-alpha,
 body.order-by-color #order-color,
 body.order-by-relevance #order-relevance {
```

These selectors must be updated as well.

```suggestion
body.order-color #order-color,
body.order-relevance #order-relevance {
```